js/script.js

```diff
@@ -93,17 +93,17 @@ function getRandomQuote () {
   return quotes[getRandomQuote];
 }
 console.log(getRandomQuote());
-
 /***
  * `printQuote` function
 ***/
 function printQuote () {
   let message = "";
   let showQuote = getRandomQuote(quotes);
+  //HI KEVIN, THIS IS WHERE I AM STUCK, SHOULD I DO A ELSE IF CLAUSE?
```

Comment:
so you need to confirm that the showQuote has a quote value....so you can console log or use debugger breakpoint...to make sure showQuote has a value....

Reply:
you need to grab the html elements that you want to change the text for and then use the quoteObject values to update their innerText